ubuntudesign/docs.vanillaframework.io
# Replace old docs.vanillaframework.io method

Use documentation-builder to build the site, as it will be going forward (even though it won't be done by this repo before too long).

QA
---

Open up `build-html` and edit it so that it says "git clone --branch develop https://github.com/vanilla-framework/vanilla-framework".

Then run the site, following the README. Check it works.
